src/app/components/home/home.component.ts

```diff
@@ -8,4 +13,20 @@ import { ChangeDetectionStrategy, Component, ViewEncapsulation } from '@angular/
   changeDetection: ChangeDetectionStrategy.OnPush
 })
 export class HomeComponent {
+  @ViewChild('mobileMenu') mobileMenu!: ElementRef;
+
+  readonly categories$: Observable<CategoryModel[]> = this._categoryService.getCategories().pipe(shareReplay(1));
+  readonly stores$: Observable<StoreModel[]> = this._storeService.getStories();
+  readonly getToKnowsUsElements$: Observable<string[]> = of(['Company', 'About', 'Blog', 'Help Center', 'Our Value']);
+
+  constructor(private _categoryService: CategoryService, private _storeService: StoreService) {
+  }
+
+  public openMobileMenu(): void {
+    this.mobileMenu.nativeElement.style = 'transform: translateX(0); visibility: visible';
```

Comment:
nope, make the subject property to keep it reactive and then in template simply use ngClass to add proper bootstrap utility css class if the value in subject is true